<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyDDosTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="domainInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>server.com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1.2.3.4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>80</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="btnGrp">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>120</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>攻击类型:</string>
    </property>
    <widget class="QRadioButton" name="icmpChk">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>94</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ICMP Flood</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="synChk">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>94</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SYN Flood</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="udpChk">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>94</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>UDP Flood</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="tcpChk">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>94</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP Flood</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="getIPBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>获取IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AtkBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>攻击</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>目标域名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>目标IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>目标端口:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
